docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,494 @@
+---
+sidebar_position: 2
+slug: /modeling/migrating/migrating-schema-1-1
+---
+
+import {
+  AuthzModelSnippetViewer,
+  CardBox,
+  CheckRequestViewer,
+  DocumentationNotice,
+  ProductConcept,
+  ProductName,
+  ProductNameFormat,
+  ReadRequestViewer,
+  RelatedSection,
+  RelationshipTuplesViewer,
+  WriteRequestViewer,
+} from '@components/Docs';
+
+# Migrating Models To Schema 1.1
+
+<DocumentationNotice />
+
+<ProductName format={ProductNameFormat.ShortForm} /> has introduced a new DSL schema version with several changes that we
+believe will make models easier to read and write, enable better tuple and model validations, and provide more options for
+optimizing the performance of different <ProductName format={ProductNameFormat.ShortForm} /> APIs.
+
+In short, we will be:
```

Comment:
This doesn't appear to be rendering with an additional newline.

<img width="446" alt="Screen Shot 2022-12-08 at 2 10 23 PM" src="https://user-images.githubusercontent.com/2899204/206568107-e1189796-b007-40a2-aa06-90f766027cca.png">

It would be nice to add a trailing newline to the paragraph before this sentence